.eslintignore

```diff
@@ -1 +1,2 @@
-jest.config.js
\ No newline at end of file
+jest.config.js
+/src/lib/interfaces.ts
```

Comment:
Oh, I see I just meant adding a `// eslint-disable-next-line @typescript-eslint/no-empty-interface` above the line of code I mentioned.  We don't want to ignore the whole file.

Reply:
Oh I see